SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/StaticSubobjectsTest/StaticSubobjectsTest.cpp

```diff
@@ -203,6 +203,8 @@ void AStaticSubobjectsTest::PrepareTest()
 	// Step 18 - In preparation for cleanup
 	MoveClientPawn(PawnSpawnLocation);
 
+	WaitForRelevancyUpdateIfInNative();
```

Comment:
Shouldn't this wait be at the start of Ajanth's test instead of the end of static? (I guess it's imaginable tests could be reordered)

couldn't we increase the time limit on the first step or whatever is failing there?

Reply:
We could, but I think I would rather just ensure that the ending state of this test is as valid as possible

Reply:
makes sense